File: src/main/java/core/gui/IRefreshable.java
// %1127326956603:plugins%
package core.gui;

/**
 * An Object of this interface can be registered in the IGUI-Interface.  The method are called, if
 * data change ( new HRF downloaded, etc. )
 */
public interface IRefreshable {
    //~ Methods ------------------------------------------------------------------------------------

    /**
     * Called, if the data changed
     */
    public void refresh();
}

File: src/main/java/core/gui/IRefreshable.kt
package core.gui

/**
 * An Object of this interface can be registered in the IGUI-Interface.
 * The <code>refresh</code> method is called, if data changes (new HRF downloaded, etc.).
 */
interface IRefreshable {
    /**
     * Called, if the data changed
     */
    fun refresh()
}
